src/main/resources/config/application.properties

```diff
@@ -25,11 +25,12 @@ spring.jpa.hibernate.ddl-auto=none
 spring.jpa.hibernate.naming-strategy=org.springframework.boot.orm.jpa.SpringNamingStrategy
 spring.jpa.properties.hibernate.cache.use_second_level_cache=true
 spring.jpa.properties.hibernate.cache.use_query_cache=false
-#spring.jpa.properties.hibernate.cache.region.factory_class=org.hibernate.cache.ehcache.SingletonEhCacheRegionFactory
+spring.jpa.properties.hibernate.cache.region.factory_class=org.hibernate.cache.jcache.JCacheRegionFactory
 spring.jpa.properties.hibernate.generate_statistics=true
 spring.jpa.properties.hibernate.search.default.directory_provider=ram
 #spring.jpa.properties.hibernate.search.default.directory_provider: filesystem
 spring.jpa.properties.hibernate.search.default.indexBase=./target/lucene
+spring.jackson.serialization.fail-on-empty-beans=false
```

Comment:
@jaykamal why was this property added?

Reply:
@dmcgihtsdo After updated cache configuration as part of SB3 Upgrade.. We have faced empty bean exception on serialization so we have added this property to resolve this issue.